app/scripts/views/settings/two_step_authentication.js

```diff
@@ -76,10 +77,23 @@ const View = FormView.extend({
     return false;
   },
 
+  _showRecoveryCodes(recoveryCodes) {
+    this.model.set('recoveryCodes', recoveryCodes);
+    this.navigate('/settings/two_step_authentication/recovery_codes', {recoveryCodes});
+  },
+
+  _replaceRecoveryCodes() {
```

Comment:
It seems like this functionality is out of place here, but I haven't played with the UI yet.

Reply:
I think this is correct because after setting up TOTP you can replace your recovery codes at anytime.